cmd/snap/cmd_info_test.go

```diff
@@ -690,8 +699,8 @@ description: |
 snap-id:      mVyGrEwiqSi5PugCwyH7WgpoQLemtTd6
 tracking:     beta
 refresh-date: 2006-01-02T22:04:07Z
-installed:    2.10 (1) 1kB disabled
-`)
+installed:    2.10 (1) 1kB disabled,blocked
```

Comment:
the repetition here vs the "health" stanza is a bit strange, not sure what we want to do about it / if anything

Reply:
it only says something for the non-healthy cases though, so I think it's fine (it calls attention to the health block that might otherwise go unnoticed)